.github/workflows/only-ci.yaml

```diff
@@ -8,8 +8,19 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ 1.13.x ]
+        os: [ ubuntu-latest ]
+    runs-on: ${{ matrix.os }}
     steps:
+      - uses: actions/checkout@master
```

Comment:
are we supposed to have 2x checkouts? (https://github.com/alexellis/k3sup/pull/283/files#diff-df63d808e4fa88b92c966c08f5d9bfacf58325695381ed547390fc52eb39360cR24)

Reply:
we already used actions/checkout@master step in [faas-netes workflow](https://github.com/openfaas/faas-netes/blob/master/.github/workflows/build.yaml), should I update it ?

Reply:
its the checkout master on below your added section im asking about, line 24-26. We checkout here, (line 17) then setup go (line 20) then re-checkout master - not sure we need both?

Reply:
done, thank you. It's my mistake, we don't need both, you're right.

Reply:
I updated files according to your review 👍

Reply:
Do we have the same issue with the arkade PR?

Reply:
Yep, I opened the same issue for the k3sup .